tests/test_particlegroup.py

```diff
@@ -34,6 +35,11 @@ def array_key(request):
     return request.param
 
 
+@pytest.fixture(params=ARRAY_KEYS)
```

Comment:
What's the difference between `array_key2` and `array_key`?

Reply:
I need it to do all combinations of two keys